app/javascript/components/application_form/preferences/AliceGriffithFields.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { map } from 'lodash'
+import { Select, Text } from 'react-form'
+
+import { buildFieldId } from './utils'
+import { typeOfProofValues } from './values'
+import Row from '~/components/atoms/Row'
+import Column from '~/components/atoms/Column'
+import FormGroup from '~/components/atoms/FormGroup'
+
+const buildTypeOfProofOptions = (values) => {
+  return map(values, (option) => (
+    { value: option, label: option }
+  ))
+}
+
+const AddressRow = ({fieldId}) => (
+  <React.Fragment>
+    <Row form>
+      <Column span={6} form>
+        <FormGroup>
+          <label htmlFor='address'>Alice Griffith Address</label>
+          <Text id='address' field={fieldId('address')}/>
+        </FormGroup>
+      </Column>
+    </Row>
+
+    <Row form>
+      <Column span={3} form>
+        <FormGroup>
+          <label htmlFor='city'>City</label>
+          <Text id='city' field={fieldId('city')}/>
+        </FormGroup>
+      </Column>
+      <Column span={3} end form>
+        <Row>
+          <Column span={6}>
+            <FormGroup>
+              <label htmlFor='state'>State</label>
+              <Text id='state' field={fieldId('state')} />
+            </FormGroup>
+          </Column>
+          <Column span={6} end>
+            <FormGroup>
+              <label htmlFor='zipCode'>Zip</label>
+              <Text id='zipCode' field={fieldId('zipCode')}/>
+            </FormGroup>
+          </Column>
+        </Row>
+      </Column>
+    </Row>
+  </React.Fragment>
+)
+
+const AliceGriffithFields = ({i, householdMembers}) => {
+  const typeOfProofOptions = buildTypeOfProofOptions(typeOfProofValues)
+  const fieldId = (field) => buildFieldId(i, field)
+
+  return (
+    <React.Fragment>
+      <Row form>
+        <Column span={3} form>
+          <label htmlFor='hhMemberOnProof'>HH Member on Proof</label>
+          <Select
+            id='hhMemberOnProof'
+            field={fieldId('naturalKey')}
+            options={householdMembers}
+            value={fieldId('naturalKey')}
+          />
+        </Column>
+        <Column span={3} form end>
+          <label htmlFor='typeOfProof'>Type of Proof</label>
```

Comment:
This works, though keep in mind that HTML is not case sensitive, so `typeOfProof` == `typeofproof`. Generally I prefer kebab case in HTML attrs for this reason, i.e. `type-of-proof`

Reply:
Also, same comment as above--we may have more than one type of proof field on the page, so better to prefix with preference identifier.